### PR TITLE
Issue/2022 exporter feature flag

### PR DESCRIPTION
**Pull request by kardan**
- [ ] **Update release notes if necessary**

### CODE REVIEW

File: exporter/index.js [hunk 1]
```diff
@@ -22,29 +22,34 @@ const validation = {
 };
 
 let sentryClient;
-const sentryIsEnabled = () => sentryClient == null;
+const sentryIsEnabled = () => !(sentryClient == null);
```
Comment (finnfiddle) [inline exporter/index.js]: splitting hairs but could be:
```const sentryIsEnabled = () => sentryClient;```

File: exporter/index.js [hunk 2]
```diff
@@ -22,29 +22,34 @@ const validation = {
...
 
-if (process.env.SENTRY_DSN == null && process.env.SENTRY_RELEASE == null
-  && process.env.SENTRY_ENVIRONMENT == null && process.env.SENTRY_SERVER_NAME == null) {
-  console.log('Sentry not configured');
-} else {
+const enableSentry = !(
```
Comment (finnfiddle) [inline exporter/index.js]: not sure this logic seems correct. should all of them be set or at least one? I think all is correct

File: exporter/index.js [hunk 3]
```diff
@@ -22,29 +22,34 @@ const validation = {
...
   sentryClient = Sentry.getCurrentHub().getClient();
-  console.log('Sentry initiated');
+} else {
+  console.log('Skipping Sentry');
 }
 
 const captureException = (error, runId = '') => {
-  console.error(`Exception captured for run ID: ${runId} -`, error);
-  if (sentryIsEnabled) Sentry.captureException(error);
+  console.error(`Exceptiqon captured for run ID: ${runId} -`, error);
```
Comment (finnfiddle) [inline exporter/index.js]: another split hair

File: exporter/index.js (hunk 2 above)
Comment (kardan) [reply exporter/index.js]: All of them should be set. Have I fooled myself?

File: exporter/index.js (hunk 3 above)
Comment (kardan) [reply exporter/index.js]: What's the issue?

File: exporter/index.js (hunk 2 above)
Comment (finnfiddle) [reply exporter/index.js]: I think if one is not equal to `null` then `enableSentry` will be `true`. Have _i_ fooled myself?

File: exporter/index.js (hunk 3 above)
Comment (finnfiddle) [reply exporter/index.js]: the spelling

File: exporter/index.js (hunk 2 above)
Comment (kardan) [reply exporter/index.js]: Yes, I think you're right. I was too focused trying to support both null and undefined & fooled myself.

File: exporter/index.js (hunk 3 above)
Comment (kardan) [reply exporter/index.js]: :-), yes that was an alternative spelling.

**Pull request merged**